Ckan UI#428

annikaLiving opened the pull request:
Added a few NA to appropriate fields, removed help text where not needed, removed html text for now.

BrandonSharratt reviewed (changes requested):
There are a few json errors, IE missing a comma on line 505 (form_placeholder) for Resource Description, and Not all of the changes were made that are required to change Pending published to pending publish

annikaLiving commented:
@BrandonSharratt i have added the missing comma and copied the master text fully for the state change. also added some NA's to some lists since we haven't decided to remove those fields yet.

adodge-bcg reviewed (approved):
Looks good to me!
